demo/psa/prov-verif-e2e/README.md

```diff
@@ -0,0 +1,162 @@
+
+In a new shell session
+
+```shell
+go install github.com/veraison/corim/cocli@demo-psa-1.0.0
+```
+
+To run provisioning test you need three parallel shell sessions.
+
+In each shell, move to the directory location where you are going to clone the GIT repo and do:
+
+```shell
+export TOPDIR=$(pwd)
+```
+
+### Steps
+
+In the first shell, clone the Veraison services repository:
+
+```shell
+git clone https://github.com/veraison/services
+```
+
+Build the services:
+
+```shell
+make -C ${TOPDIR}/services
+```
+
+Start the REST API frontend:
+
+```shell
+( cd ${TOPDIR}/services/provisioning/cmd && ./provisioning-service )
+```
+
+In another shell create the KV stores:
+
+```shell
+( cd ${TOPDIR}/services/vts/cmd && ../test-harness/init-kvstores.sh )
+````
+
+Then start the VTS service:
+
+```shell
+( cd ${TOPDIR}/services/vts/cmd && ./vts-service )
+```
+
+In the third shell
+
+```shell
+git clone https://github.com/veraison/docs
+```
+
+Move to the docs/demo/psa/prov-verif-e2e directory:
+
+```shell
+cd ${TOPDIR}/docs/demo/psa/prov-verif-e2e
+```
+
+Ship the single CORIM that contains PSA reference values and trust anchor from data/cbor folder using following command
+
+```shell
+cocli corim submit --corim-file=data/cbor/corim-full.cbor --api-server="http://localhost:8888/endorsement-provisioning/v1/submit" --media-type="application/corim-unsigned+cbor; profile=http://arm.com/psa/iot/1"
+```
+
+The REST frontend should return a success status.
+
+If so, you can inspect the KV stores to check what has been generated:
+
+* Verification keys:
+
+```shell
+sqlite3 ${TOPDIR}/services/vts/cmd/ta-store.sql 'select distinct vals from kvstore' | jq .
+```
+
+* Reference values:
+
+```shell
+sqlite3 ${TOPDIR}/services/vts/cmd/en-store.sql 'select distinct vals from kvstore' | jq .
+```
+
+### Cleanup
+
+Use Ctrl-C to stop the REST provisioning frontend in shells 1 (as one 
+would not need provisioning service once the Reference Values and
+Endorsements are now provisioned into Verification service)
+
+## Verification pipeline setup
+
+If not already done, ensure the verification service is running:
+
+```shell
+( cd ${TOPDIR}/services/verification/cmd && ./verification-service )
+```
+### Exchanging evidence with the Verifier
+
+In a new shell session
+
+```shell
+export TOPDIR=$(pwd)
+```
+
+```shell
+go install github.com/veraison/evcli@demo-psa-1.0.0
+```
+
+```shell
+cd ${TOPDIR}/docs/demo/psa/prov-verif-e2e
+```
+
+* Verifying Evidence as a Relying party with the Veraison Verifier
+
+1. First create Evidence using supplied templates
+
+```shell
+evcli psa create -c data/templates/psa-claims-profile-2.json -k data/keys/ec-p256.jwk --token=psa-evidence.cbor
+```
+` >> "psa-evidence.cbor" successfully created `
+
+2. Exchange Evidence with the Verification Service
+
+* Verifying as a Relying Party
+
+```shell
+evcli psa verify-as relying-party --api-server=http://localhost:8080/challenge-response/v1/newSession --token=psa-evidence.cbor
+```
+
```

Comment:
Evcli does not print any result of the verification operation (success or failure) here.  Wouldn't it be much better if evcli prints the result of verification rather than user traverse the console(s) and checks the logs from vts .?? Especially for someone like me who get often lost in fishing out the right console ...

Reply:
Yes, we will be working on this issue: Please see https://github.com/veraison/evcli/issues/12